newIDE/app/src/MainFrame/EditorContainers/HomePage/TeamSection/NewTeamGroupNameField.js

```diff
@@ -0,0 +1,107 @@
+// @flow
+
+import * as React from 'react';
+import { Line } from '../../../../UI/Grid';
+import IconButton from '../../../../UI/IconButton';
+import CheckIcon from '../../../../UI/CustomSvgIcons/Check';
+import CrossIcon from '../../../../UI/CustomSvgIcons/Cross';
+import {
+  shouldValidate,
+  shouldCloseOrCancel,
+} from '../../../../UI/KeyboardShortcuts/InteractionKeys';
+import TextField from '../../../../UI/TextField';
+import { Trans, t } from '@lingui/macro';
+
+type Props = {|
+  onValidateGroupName: ({| name: string |}) => Promise<void>,
+  onDismiss: () => void,
+|};
+
+const NewTeamGroupNameField = ({ onValidateGroupName, onDismiss }: Props) => {
+  const [isLoading, setIsLoading] = React.useState<boolean>(false);
+  const [errorText, setErrorText] = React.useState<?React.Node>(null);
+  const [groupName, setGroupName] = React.useState<string>('');
+
+  const onFinishEditingName = React.useCallback(
+    async () => {
+      const cleanedGroupName = groupName.trim();
+      if (!cleanedGroupName) {
+        setErrorText(<Trans>Group name cannot be empty.</Trans>);
+        return;
+      }
+      setIsLoading(true);
+      try {
+        await onValidateGroupName({ name: cleanedGroupName });
+        setGroupName('');
+        setErrorText(null);
+        onDismiss();
+      } catch (error) {
+        console.error(error);
```

Comment:
Either remove the log or add the context `console.error("Error while xxx", error)`